acoular/calib.py

```diff
@@ -12,16 +12,24 @@
 # imports from other packages
 from os import path
 
-from numpy import array
-from traits.api import CArray, CLong, File, HasPrivateTraits, Property, cached_property, on_trait_change
+from numpy import array, newaxis
+from traits.api import CArray, CLong, File, Property, cached_property, on_trait_change
+
+from .base import TimeOut
 
 # acoular imports
 from .internal import digest
 
 
-class Calib(HasPrivateTraits):
-    """Container for calibration data in `*.xml` format.
+class Calib(TimeOut):
```

Comment:
Calib should be better derived from `InOut` instead from `TimeInOut`

This enables the class to be used in the frequency domain. This should also be mentioned in the docstring (ideally with a short code snippet on how to use this class).